### PR TITLE
Vtcca-5639 error message issues occupancy page

### PR DESCRIPTION
**Pull request by michjanowicz**
Passed branch testing

### CODE REVIEW

File: test/controllers/propertyLinking/ClaimPropertyOccupancyControllerSpec.scala [hunk 1]
```diff
@@ -117,13 +117,57 @@ class ClaimPropertyOccupancyControllerSpec extends VoaPropertyLinkingSpec with S
...
         error.text() shouldBe
-          s"Date of ${if (isAgent) "your client's " else ""}last day as " +
+          s"Date of ${if (isAgent) "your client's " else "your "}last day as " +
             s"${if (cap == Owner) "owner" else if (cap == Occupier) "occupier" else "owner and occupier"} " +
             "of the property must be after " +
             s"${d._1.getDayOfMonth} ${d._1.getMonth.toString.toLowerCase.capitalize} ${d._1.getYear}"
       }
     }
   }
+  "The claim occupancy page" should "validate that the end date of ownership is not in the future" in {
```
Comment (joeyymoore) [inline test/controllers/propertyLinking/ClaimPropertyOccupancyControllerSpec.scala:127]: We need some sort of welsh tests for this as-well, especially with the ticket saying that something has been incorrectly translated into Welsh

File: test/controllers/propertyLinking/ClaimPropertyOccupancyControllerSpec.scala [hunk 2]
```diff
@@ -117,13 +117,57 @@ class ClaimPropertyOccupancyControllerSpec extends VoaPropertyLinkingSpec with S
...
+          "stillOccupied"          -> "false",
+          "lastOccupiedDate.day"   -> endDate.getDayOfMonth.toString,
+          "lastOccupiedDate.month" -> endDate.getMonthValue.toString,
+          "lastOccupiedDate.year"  -> endDate.getYear.toString
+        ))
+        status(result) shouldBe BAD_REQUEST
+        val doc = Jsoup.parse(contentAsString(result))
+        val error = doc.getElementsByAttributeValue("href", "#lastOccupiedDate-day")
+        error.text() shouldBe
+          s"Date ${if (isAgent) "your client " else "you "}became " +
```
Comment (joeyymoore) [inline test/controllers/propertyLinking/ClaimPropertyOccupancyControllerSpec.scala:146]: i know the forAll tests all the combinations of the scenarios, but for me id rather just have different tests. Makes it easier to work out whats being tested when and what the messages are

**Comment by michjanowicz** [comment]
tested on branch LGTM 👍

**Comment by zachhall1234** [comment]
Looks good to me 👌🏼

**Pull request merged**